package.json

```diff
@@ -23,7 +22,8 @@
     "lint": "eslint --fix .",
     "lint:report": "eslint --ext .ts --output-file eslint_report.json --format json .",
     "deploy": "hardhat deploy",
-    "docgen": "yarn build && typedoc && hardhat docgen"
+    "docgen": "yarn build && typedoc && hardhat docgen",
+    "test:matic": "cross-env CHAINID=137 hardhat test test/SynthsSdk.test.ts"
```

Comment:
Just formal. I suggest to move `test:matic` up to line 18 where the other test scripts are declared.